direct/src/dist/commands.py

```diff
@@ -26,14 +26,16 @@
 from . import pefile
 from .icon import Icon
 import panda3d.core as p3d
-
+import warnings
 
 if sys.version_info < (3, 0):
     # Warn the user.  They might be using Python 2 by accident.
-    print("=================================================================")
-    print("WARNING: You are using Python 2, which has reached the end of its")
-    print("WARNING: life as of January 1, 2020.  Please upgrade to Python 3.")
-    print("=================================================================")
+    # Do we want to leave this as this type of warning or should we switch to warnings class?
+    warnings.warn("=================================================================\n \
+                 WARNING: You are using Python 2, which has reached the end of its\n \
+                 WARNING: life as of January 1, 2020.  Please upgrade to Python 3.\n \
+                 =================================================================",
+                 DeprecationWarning)
```

Comment:
I think this should be a FutureWarning, since it needs to be displayed by default to end-users, especially considering that DeprecationWarning doesn't display by default in Python 2.

That said, I'm not sure why we still have this warning because we no longer support Python 2 to begin with; there should probably be an error message here.  That could also happen in a separate change though.

Reply:
This was updated to a FutureWarning.

Reply:
If I'm not mistaken, the indents are included in the string here?  You could avoid this by putting quotation marks around each line.